subtests/docker_cli/start/start.py

```diff
@@ -53,10 +53,8 @@ class start_base(SubSubtest):
     def initialize(self):
         super(start_base, self).initialize()
         config.none_if_empty(self.config)
-        dc = DockerContainers(self)
-        self.sub_stuff["conts_obj"] = dc
-        dc = DockerContainersRunOnly(self)
-        self.sub_stuff["con_ro_obj"] = dc
```

Comment:
R0204: 58,8: start_base.initialize: Redefinition of dc type from dockertest.containers.DockerContainers to start.DockerContainersRunOnly